distribution/docker/build.gradle

```diff
@@ -91,20 +98,79 @@ void addCopyDockerfileTask(final boolean oss) {
   }
 }
 
+Closure createAndSetWritable = {location ->
+  File file = file(location);
+  file.mkdirs()
+  file.setWritable(true, false)
+}
+
+task copyKeystore(type: Sync) {
+  from project(':x-pack:plugin:core')
+          .file('src/test/resources/org/elasticsearch/xpack/security/transport/ssl/certs/simple/testnode.jks')
+  into "${buildDir}/certs"
+  doLast {
+    file("${buildDir}/certs").setReadable(true, false)
+    file("${buildDir}/certs/testnode.jks").setReadable(true, false)
+  }
+}
+
 preProcessFixture {
   dependsOn taskName("copy", true, "DockerContext")
   dependsOn taskName("copy", true, "Dockerfile")
   dependsOn taskName("copy", false, "DockerContext")
   dependsOn taskName("copy", false, "Dockerfile")
+  dependsOn copyKeystore
+  doLast {
+    // tests expect to have an empty repo
+    project.delete("${buildDir}/repo")
```

Comment:
FYI, you can call `project.delete()` with multiple paths. Ex: `project.delete("${buildDir}/repo", "${buildDir}/oss-repo")`. It'd be convenient if we made `createAndSetWritable` _also_ accept `Object...` as an argument as well to support the same convenience.